src/sentry/incidents/models/alert_rule.py

```diff
@@ -77,6 +78,28 @@ class AlertRuleStatus(Enum):
     DISABLED = 5
 
 
+class AlertRuleDetectionType(models.IntegerChoices):
```

Comment:
I'm using a new (to us) thing here to avoid situations like https://github.com/getsentry/sentry/blob/master/src/sentry/incidents/models/alert_rule.py#L391-L409 where we reference an enum but require a comment to know which one.

Django docs are here: https://docs.djangoproject.com/en/5.0/ref/models/fields/#field-choices-enum-types
and we do have _one_ instance of this being used in Sentry today https://github.com/getsentry/sentry/blob/master/src/sentry/models/dashboard_widget.py#L161-L182

Reply:
I could be convinced to change this to "static", "percent", and "dynamic" which would require a backfill, but could be done. The thinking here was just that it's a static threshold in the sense that it's user defined or it's dynamic in the sense that they've chosen anomaly detection.

Reply:
Nice! Great find!

If we're gonna name these string values anyways, i think i'm down to make these `models.TextChoices`
makes it easier to see what the selection is when sifting through db data in redash or something

Reply:
I'm also strongly 👍  on being explicit between static and percent.
Otherwise this is effectively a boolean value.

Static threshold to me means the _threshold_ is static - not that i've chosen a static value